### PR TITLE
DShop page testimonial update

### PR DESCRIPTION
**Pull request by ZaHuPro**
Add testimonials from Ternio and Kyber and remove the photo in the Contentos testimonial

https://docs.google.com/document/d/1cr3txXJ2ynM6RbxAPiMOA91Nz9d4kp-X9AHc2-c5D-Y/edit

### CODE REVIEW

**Comment by ZaHuPro** [comment]
screenshot-localhost_5000-2021 01 16-23_00_50
Is it okay, If it is more than one row?

**Comment by ZaHuPro** [comment]
- Updated with the slider
- I have added external dependencies

screenshot-localhost_5000-2021 01 22-00_18_52 (1)

screenshot-localhost_5000-2021 01 22-00_18_52

File: views/web_views.py [hunk 1]
```diff
@@ -606,6 +606,8 @@ def assets_all_styles():
         concat_asset_files(
             [
                 "static/css/vendor-bootstrap-4.0.0-beta2.css",
+                "static/css/vendor-owl.theme.default.min.css",
+                "static/css/vendor-owl.carousel.min.css",
```
Comment (shahthepro) [inline views/web_views.py:610]: I'm not sure what's our take on adding thrid-party deps to the website. But think it's fine. 
Tagging @joshfraser @sparrowDom for their opinion.

**Review by shahthepro — APPROVED** [review]
LGTM

File: views/web_views.py (hunk 1 above)
Comment (sparrowDom) [reply views/web_views.py:610]: If it is working fine I don't have a problem with it

**Pull request merged**